lua/weapons/weapon_brap/shared.lua

```diff
@@ -0,0 +1,81 @@
+SWEP.PrintName = "BRRRAAAPPP"
+
+SWEP.Slot = 0
+
+SWEP.WorldModel = ""
+SWEP.ViewModel = ""
+SWEP.Primary.Ammo = "none"
+SWEP.Secondary.Ammo = "none"
+SWEP.Weight = 0
+SWEP.AutoSwitchTo = false
+SWEP.DrawCrosshair = false
+if SERVER then 
+	timer.Create("Braaaaap",1.7,0,function()
+		FunnyFart = player.GetHumans()
+		
+		for k,v in pairs(FunnyFart) do
+			if v:HasWeapon("weapon_brap") then
+				if math.random() < 0.05 then
+						v:GetWeapon("weapon_brap"):PrimaryAttack()
```

Comment:
won't this keep happening forever?

Reply:
Yes it's based on the ebola logic. The SWEP deletes itself once used. this gives the SWEP with a chance weighted by beans eaten and then clears the beans eaten counter. It also might randomly fire the SWEP for you if you hold onto it for too long